buku

```diff
@@ -3134,84 +3134,11 @@ n: don't add parent folder as tag
             index: Optional[int] = None,
             url: Optional[str] = None,
             shorten: bool = True) -> Optional[str]:
-        """Shorten a URL using Google URL shortener.
+        """Shorten/expand a URL using the tny.im service.
 
-        Parameters
-        ----------
-        index : int, optional (if URL is provided)
-            DB index of the bookmark with the URL to shorten. Default is None.
-        url : str, optional (if index is provided)
-            URL to shorten.
-        shorten : bool
-            True to shorten, False to expand. Default is False.
-
-        Returns
-        -------
-        str
-            Shortened url on success, None on failure.
+        Tny.im is no longer available; don't use this method.
```

Comment:
The rest of the doc is no longer useful, so I removed it (replacing with an appropriate message).